website/docs/tutorials/08-cairo-1-support/01-project-initialization.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: Project initialization
+---
+
+# Project initialization
+
+## Creating a project
+To create a new Protostar project with cairo1 support, you will need to run the `protostar init-cairo1` command followed by the name of your project.
+
+After running the command, the following structure will be generated:
+
+```
+./your-project-name
+|-- protostar.toml
+|-- src
+|   |-- cairo_project.toml
+|   |-- lib.cairo
+|   `-- main.cairo
+`-- tests
+    `-- test_main.cairo
+```
+
+This template contains:
+
+- `src` directory
+    - `cairo_project.toml` which is needed for compilation 
+    - `main.cairo` file with one function definition
+    - `lib.cairo` file which defines the module
+- `test` directory
+    - Single test file with one test for function defined in `main.cairo`
+- `protostar.toml` containing information about the project
```

Comment:
```suggestion
- `src` – a directory for the src module
    - `main.cairo` – a module source file
    - `lib.cairo` – a module definition file
    - `cairo_project.toml` – a module configuration file
- `tests` – a directory for tests
    - `test_main.cairo` – a test file for the src module (tests the code from `main.cairo`)
- `protostar.toml` – a project configuration file
```

Reply:
We can also consider adding description for users on how to create modules.

Maybe renaming src directory to e.g. `template_module` will give be a good idea – the name `src` suggests that 
all of the source code should be put in this folder. 

We could also put it one level deeper – `src/template_module/`. That way users won't get lost and will just follow the pattern to add another modules into the `src` folder.

Reply:
Well, this doc is about project **initialization**, so playing with modules feels a bit out of scope here. I think we should approach this in a more specific (not generic) way and just tell people how to set up a quick project so they can write code.

Reply:
Okay, agree with that.

Reply:
I think we could discuss creating a doc for how to create a cairo1 module. Although i would much rather peek at the cairo1 docs first to see their approach. We could schedule that for the next milestone.

Reply:
#1591